applications/vanilla/views/discussion/helper_functions.php

```diff
@@ -130,17 +136,17 @@ function writeComment($Comment, $Sender, $Session, $CurrentOffset) {
                         ?>
                         <?php
                         // Include source if one was set
-                        if ($Source = val('Source', $Comment))
+                        if ($Source = val('Source', $Comment)) {
                             echo wrap(sprintf(t('via %s'), t($Source.' Source', $Source)), 'span', array('class' => 'MItem Source'));
```

Comment:
You previously changed the long form array() to the shorter version [].
You may want to do it everywhere.